pkg/storage/trb_request_attendee.go

```diff
@@ -50,23 +51,48 @@ func (s *Store) CreateTRBRequestAttendee(ctx context.Context, attendee *models.T
 
 // UpdateTRBRequestAttendee updates a TRB request attendee record in the database
 func (s *Store) UpdateTRBRequestAttendee(ctx context.Context, attendee *models.TRBRequestAttendee) (*models.TRBRequestAttendee, error) {
-	const sql = `
+	// return attendee, nil
+	stmt, err := s.db.PrepareNamed(`
 		UPDATE trb_request_attendees
-		SET
+		SET id = :id,
```

Comment:
Do we want to expose the ability to update the ID of a row? This seems like it could open up the possibility for a bug depending on how this function is called (but as-is, don't think it actually causes any issues)

Reply:
I agree, it's non-blocking because it isn't actually going to update the id because of the `WHERE` clause, but there isn't a reason to set `id` here.

Reply:
@ClayBenson94 removed ID from this